src/rust/lib/enso-prelude/src/lib.rs

```diff
@@ -220,7 +220,7 @@ impl<T:Default> Default for CloneCell<T> {
 // =================
 
 #[derive(Debug)]
-pub struct CloneRefCell<T> {
+pub struct CloneRefCell<T:?Sized> {
```

Comment:
ugh, this is stupid :( Why Rust just cannot infer it ...

Reply:
The funny thing is that I don't need it anymore. Shall I revert this then?